modules/promotions/admin-menu-items/go-pro-promotion-item.php

```diff
@@ -21,7 +26,9 @@ public function get_parent_slug() {
 	}
 
 	public function get_label() {
-		return esc_html__( 'Upgrade', 'elementor' );
+		$upgrade_text = __( 'Upgrade', 'elementor' );
+		$promotion['upgrade_text'] = $upgrade_text;
+		return apply_filters( 'elementor/admin_menu/promotion/upgrade_text', $promotion )['upgrade_text'] ?? $upgrade_text;
```

Comment:
This requested change to have two filters here is not ideal from an abstraction point of view. The filter interface exposed to a dev mutating promotions should be as uniform as possible to avoid confusion. Promotions always have one or more of the same elements i.e. title, description, image, upgrade_text, upgrade_url.